R/start_addin.R

```diff
@@ -0,0 +1,85 @@
+#' Addin for starting develop mode.
+#'
+#' Allows users to see list of orderly reports and choose one to open in
+#' development mode.
+#'
+#' @export
+start_addin <- function() {
```

Comment:
When I selected "internal-2017-population-TUV-MHL" for some reason "description-of-impact-methods" was loaded (with the table filtered). Is the js here working as expected?

Reply:
Fixed now - that has tidied up the status code too (removing the status df from gadget wide state)